ui/app/routes/datasets/$dataset_name/datapoint/$id/datapointOperations.server.ts

```diff
@@ -40,6 +45,50 @@ function transformOutputForTensorZero(
   }
 }
 
+/**
+ * Transforms a chat datapoint for submission to the TensorZero client.
+ * Generates a new UUID and marks the datapoint as custom.
+ */
+function transformChatDatapoint(datapoint: ParsedChatInferenceDatapointRow): Datapoint {
```

Comment:
can we add a more detailed name `transformChatDatapointForUpdate` so it is a little more clear what this is?